install_config/storage_examples/azure_blob_docker_registry_example.adoc

```diff
@@ -0,0 +1,101 @@
+[[azure-blob-docker-registry]]
+= Configuring Azure Blob Storage for Integrated Docker Registry
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap:
+
+toc::[]
+
+[[azure-blob-docker-registry-overview]]
+== Overview
+This topic reviews how to configure Azure Blob Storage for OpenShift integrated Docker Registry.
```

Comment:
In the section which I link to it is listed as 'Integrated Docker registry' https://docs.openshift.com/container-platform/latest/install_config/registry/deploy_registry_existing_clusters.html#deploy-registry